tests/python/pytorch/graphbolt/impl/test_torch_based_feature_store.py

```diff
@@ -219,10 +220,13 @@ def test_torch_based_pinned_feature(dtype, idtype, shape):
     test_tensor_cuda = test_tensor.cuda()
 
     feature = gb.TorchBasedFeature(tensor)
-    feature.pin_memory_()
+    if in_place:
+        feature.pin_memory_()
 
-    # Check if pinning is truly in-place.
-    assert feature._tensor.data_ptr() == tensor.data_ptr()
+        # Check if pinning is truly in-place.
+        assert feature._tensor.data_ptr() == tensor.data_ptr()
+    else:
+        feature = feature.to("pinned")
```

Comment:
What's the difference between `pin_memory` and `to("pinned")`?

Reply:
One pins in place without changing underlying buffer, the other moves to pinned memory using torch.Tensor.pin_memory()